config.yml

```diff
@@ -1020,6 +1020,15 @@ nodes:
 
           `foo #{bar} baz`
           ^^^^^^^^^^^^^^^^
+  - name: KeywordHashNode
+    child_nodes:
+      - name: elements
+        type: node[]
+    comment: |
+      Represents a hash literal without an opening and closing braces.
```

Comment:
Nit `Represents a hash literal without an opening and closing braces.` -> `Represents a hash literal without opening and closing braces.`